Service/src/main/java/org/apidb/apicommon/service/services/jbrowse/JBrowseService.java

```diff
@@ -240,15 +240,18 @@ public Response getJbrowseChIPSeqTracks(@PathParam("organismAbbrev") String orga
     public Response getJBrowseStore(@QueryParam("data") String data,
                                     @HeaderParam("Range") String fileRange) throws WdkModelException {
 
-        String projectId = getWdkModel().getProjectId();
         String buildNumber = getWdkModel().getBuildNumber();
-        String webservicesDir = getWdkModel().getProperties().get("WEBSERVICEMIRROR");
+        String udDir = getWdkModel().getProperties().get("UD_DIR");
+        // New WDK property: UserDatasetFiles. This is located in /var/www/Common/userDatasets
 
         String path = checkPath(
-            webservicesDir + "/" +
-            projectId + "/" +
-            "build-" + buildNumber + "/" +
-            data);
+            udDir + "/" +
+                "build-" + buildNumber + "/" +
```

Comment:
Curious why we have a build-xx dir in a user dataset dir?  Feel like UDs, once installed, would span builds unless they become invalid for some reason and then they would get uninstalled or patched.

Reply:
e.g. how does the installer know which build to put these in?  Are you forcing its config change each release?   If so, then they'll all need to be reinstalled like we do now right?  I thought in blue/green we were hoping to only install new datasets, and into the newly QA dir (recently swapped from prod).

Reply:
Yes, in the short-term we don't really have a good way to avoid this. Once we actually have lifecycle management for blue/green instances we'll likely replace the build number part of the directory with "blue" or "green" or whatever we decide to call the instances.